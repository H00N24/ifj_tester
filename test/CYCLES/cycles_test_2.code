// Rozsirenie CYCLES

class Main
{
	static void run()
	{
		for(int i = 0; i < 5; i = i + 1)
		{
			ifj16.print(i + "\n");
			for(int j = 10; j > 5; j = j - 1)
			{
				ifj16.print(j + "\n");
			}
		}
	}
}
