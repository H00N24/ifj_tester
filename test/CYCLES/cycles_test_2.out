0
10
9
8
7
6
1
10
9
8
7
6
2
10
9
8
7
6
3
10
9
8
7
6
4
10
9
8
7
6
